File: Python/ExampleTests/production/AddressComponents.py
class AddressComponents:
    def __init__(self, zip: str = '', street_name: str = '', city: str = '', pre_direction: str = '', suffix_direction: str = '', state: str = '', suffix_type: str = ''):
        self.zip = zip
        self.street_name = street_name
        self.city = city
        self.pre_direction = pre_direction
        self.suffix_direction = suffix_direction
        self.state = state
        self.suffix_type = suffix_type

File: Swift/ExampleTests/production/AddressComponents.swift
public struct AddressComponents: Equatable {
    public let zip: String
    public let streetName: String
    public let city: String
    public let preDirection: String
    public let suffixDirection: String
    public let state: String
    public let suffixType: String

    public init(zip: String, streetName: String, city: String, preDirection: String, suffixDirection: String, state: String, suffixType: String) {
        self.zip = zip
        self.streetName = streetName
        self.city = city
        self.preDirection = preDirection
        self.suffixDirection = suffixDirection
        self.state = state
        self.suffixType = suffixType
    }
}
